components/tabs/style/index.less

```diff
@@ -183,6 +183,14 @@
       cursor: not-allowed;
```

Comment:
why not write after this line?

Reply:
sorry, I'm not familiar with less, do you have any good suggestions for me?

like this? I think the readability is not good.

```css
  &&-disabled {
      color: @disabled-color;
      cursor: not-allowed;
      
      .@{tab-prefix-cls}-tab-btn,
      .@{tab-prefix-cls}-tab--remove {
        &:focus,
        &:active {
          color: @disabled-color;
        }
      }
    }
```

Reply:
How about `pointer-events: none`?